public/js/app/util/links.js

```diff
@@ -40,7 +46,8 @@ function programmeWebLink(programmeCode, language) {
 }
 
 function appendix1Link(programmeCode, term) {
-  return `/student/kurser/program/${programmeCode}/${term}/kurslista`
+  const baseUrl = possiblyChangeBaseUrl()
+  return `${baseUrl}/student/kurser/program/${programmeCode}/${term}/kurslista`
```

Comment:
```suggestion
  return pageLink(`/student/kurser/program/${programmeCode}/${term}/kurslista`)
```
I think this is the only place you use `possiblyChangeBaseUrl` outside of `domain/links.js`. So test if it is possible to use `pageLink()` here as well.

Reply:
This was a great comment ⭐️ I made the required changes and tested it in my local environment; everything looks good.